airbyte-integrations/connectors/source-apple-search-ads/acceptance-test-config.yml

```diff
@@ -0,0 +1,30 @@
+# See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-apple-search-ads:dev
+acceptance_tests:
+  spec:
+    tests:
+      - spec_path: "source_apple_search_ads/spec.yaml"
+  connection:
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
+  discovery:
+    tests:
+      - config_path: "secrets/config.json"
+  basic_read:
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        empty_streams: []
+  incremental:
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        future_state_path: "integration_tests/abnormal_state.json"
```

Comment:
```suggestion
        future_state:
          future_state_path: "integration_tests/abnormal_state.json"
```
There was a typo in the code generator (now fixed), please make this minor change.

Reply:
I'm finally able to run the tests! Thank you 🙌 
I'm doing some changes as the CDK evolved since the PR was opened, and I'll post the results soon

Reply:
@alafanechere I've been able to run the tests on a 2days lookback window:
![image](https://user-images.githubusercontent.com/69161287/214016790-6d6a8ffe-1d2e-4a19-bea4-e15ac94242ca.png)

However I'm not able to run them in a 30days lookback window as it times out eventually (lots of data to retrieve on our company account) - even if the `start_date` is set to today. I could increase the timeout but I don't think the acceptance tests should run for days? 😬 I've mentioned this issue in the Slack channel [here](https://airbytehq.slack.com/archives/C027KKE4BCZ/p1674215189136759) but could not find any workaround yet